\hypertarget{classneo4j_1_1exceptions_1_1_result_consumed_error}{}\doxysection{neo4j.\+exceptions.\+Result\+Consumed\+Error Class Reference}
\label{classneo4j_1_1exceptions_1_1_result_consumed_error}\index{neo4j.exceptions.ResultConsumedError@{neo4j.exceptions.ResultConsumedError}}


Inheritance diagram for neo4j.\+exceptions.\+Result\+Consumed\+Error\+:
% FIG 0


Collaboration diagram for neo4j.\+exceptions.\+Result\+Consumed\+Error\+:
% FIG 1


\doxysubsection{Detailed Description}
\begin{DoxyVerb} Raised when trying to access records after the records have been consumed.
\end{DoxyVerb}
 

The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
venv/\+Lib/site-\/packages/neo4j/exceptions.\+py\end{DoxyCompactItemize}
